Makefile

```diff
@@ -386,3 +390,22 @@ verify-generated: bundle generate
 
 install-git-hooks:
 	git config core.hooksPath .githooks	
+
+GOLANGCI_LINT := $(shell command -v golangci-lint 2> /dev/null)
+GOLANGCI_LINT_LOCAL_VERSION := $(shell command golangci-lint --version | awk '{print $$4}')
+golangci-lint:
+ifdef GOLANGCI_LINT
```

Comment:
can we have the check like we do for `operator-sdk` and `opm` ? I'd also omit the version check/postpone to later

Reply:
done